go.mod

```diff
@@ -225,32 +237,38 @@ require (
 	go.opentelemetry.io/otel/trace v1.21.0 // indirect
 	go.uber.org/atomic v1.10.0 // indirect
 	go.uber.org/multierr v1.9.0 // indirect
+	golang.org/x/mod v0.17.0 // indirect
 	golang.org/x/oauth2 v0.16.0 // indirect
-	golang.org/x/sync v0.12.0 // indirect
-	golang.org/x/sys v0.31.0 // indirect
-	golang.org/x/term v0.30.0 // indirect
+	golang.org/x/sync v0.6.0 // indirect
+	golang.org/x/sys v0.18.0 // indirect
+	golang.org/x/term v0.18.0 // indirect
 	golang.org/x/time v0.5.0 // indirect
+	golang.org/x/tools v0.15.0 // indirect
 	google.golang.org/api v0.155.0 // indirect
 	google.golang.org/appengine v1.6.8 // indirect
 	google.golang.org/genproto v0.0.0-20240123012728-ef4313101c80 // indirect
 	google.golang.org/genproto/googleapis/rpc v0.0.0-20240125205218-1f4bbc51befe // indirect
 	gopkg.in/ini.v1 v1.67.0 // indirect
-	gopkg.in/natefinch/npipe.v2 v2.0.0-20160621034901-c1b8fa8bdcce // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 	nhooyr.io/websocket v1.8.6 // indirect
 	pgregory.net/rapid v1.1.0 // indirect
+	rsc.io/tmplfunc v0.0.3 // indirect
 	sigs.k8s.io/yaml v1.4.0 // indirect
 )
 
 replace (
 	cosmossdk.io/api => cosmossdk.io/api v0.3.1
 
 	github.com/CosmWasm/wasmd => github.com/NibiruChain/wasmd v0.44.0-nibiru
+	// github.com/cockroachdb/pebble: Has to be pinned for compatibilit with
+	// go-ethereum/ethdb, which uses pebble imports
+	github.com/cockroachdb/pebble => github.com/cockroachdb/pebble v0.0.0-20230928194634-aa077af62593
```

Comment:
I'm 90% sure this will break nodes that run with PebbleDB. I ran into this issue last year (https://nibiruchain.slack.com/archives/C03DYE815BK/p1724088146798449?thread_ts=1723844930.557689&cid=C03DYE815BK), which is why we're focused on upgrading cometbft (in addition to Cosmos SDK).

Reply:
I can see about whether this is fixed in an even newer version of geth and come back. It may have been addressed in v1.14 or 1.15

Reply:
Update: v1.14 does use a newer pebble (`github.com/cockroachdb/pebble v1.1.2`). This has been fixed by 7c52ee37e524fe2ad8c21faedb645f3c0ce74ffe